src/frontend/app.ts

```diff
@@ -102,7 +103,8 @@ export default (octokitApp: App): Express => {
 			maxAge: 30 * 24 * 60 * 60 * 1000, // 30 days
 			signed: true,
 			sameSite: "none",
-			secure: true
+			secure: true,
+			httpOnly: false
```

Comment:
By default, session cookies can only be modified by the server, but we needed the frontend to be able to set the `jiraHost` cookie to be saved in session.